### ISSUE TITLE
Local.yaml configuration

### ISSUE THREAD

**Issue by CptCurk** [YunoHost-Apps/peertube_ynh#413]
**Hello,**

Thank you for the ability to get easily peertube on Yuhohost. 

I don't understand how that work... 

First, I duplicate the `production.yaml` rename into `local.yaml`, gave the same rights, user & group. But nothing seems to work, and it's still using the `production.yaml` (which creat a local-production.json... ^^) so how to not get my setup erased at the next update of peertube ? 

Or is this json file already something which will not be erased by the next update ? 

thank you !

**Comment by ericgaspar** [comment]
The `production.yaml` file will get regenerated with the upgrade to PeerTube 6.0.2. There is a lot of changes upstream in that file that we nee to add.

**Comment by CptCurk** [comment]
I basicly update with the testing branch to get my video files working again. 
Any preset changed. I quite don't understand what's happenin now with presets.

**Comment by ericgaspar** [comment]
What problem are you facing with presets? did you manually change `production.yaml`? what setting do you want to change?

**Comment by CptCurk** [comment]
no, what I want to say is that I don't get any trouble with my preset.

But I don't understand why need to write this file local.yaml ! Because even with update, everything seems to work on my peertube ! And when I wrote production.yaml (inside the file using nano) nothing changed, I had peertube CLI open, may be it overwrite everything instantly ? 

Anyway, everything's looks good now, without using local.yaml !

I just don't understand why it's writing to use local.yaml instead of production. That's it ! ^^'

**Comment by ericgaspar** [comment]
> And when I wrote production.yaml (inside the file using nano) nothing changed

did you restart PeerTube service after modifying `production.yaml` file?

**Comment by CptCurk** [comment]
okay. That's it, I now understand. Thank you ^^' realy simple, but need to think about it. 

But everything seems to be working only with the CLI. I don't need command line as simple admin user. 

Thank you !
